GherkinSyncTool/GherkinSyncTool/Models/FeatureFile.cs

```diff
@@ -8,20 +8,19 @@ namespace GherkinSyncTool.Models
     public class FeatureFile : IFeatureFile
     {
         public GherkinDocument Document { get; init; }
-
+        public string AbsolutePath { get; }
+        public string RelativePath { get; }
+        
         public FeatureFile(GherkinDocument document, string path)
         {
             if (!new FileInfo(path).Exists)
                 throw new DirectoryNotFoundException($"File {path} not found");
-            
             Document = document;
             AbsolutePath = Path.GetFullPath(path);
-            RelativePath = Path.GetRelativePath(
-                Directory.GetParent(ConfigurationManager.GetConfiguration().BaseDirectory).FullName, 
-                AbsolutePath);
+            var baseDirectory = new DirectoryInfo(ConfigurationManager.GetConfiguration().BaseDirectory);
+            var relativeToDirectory = baseDirectory?.Parent?.FullName ?? 
+                                      throw new FileNotFoundException($"Base directory {baseDirectory} does not have a parent. Feature file creation is not possible");
```

Comment:
The tool is not responsible for creating feature files. Let's keep only "The base directory {baseDirectory} does not have a parent."